### ISSUE TITLE
When I try to open a modal from another modal, the modal exits

### ISSUE THREAD

**Issue by skamensky** [cortezaproject/corteza#1285]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Version of Corteza

2023.3.3

### Current Behavior

Layout:

A record list that when clicked opens a modal.
That modal has tabs. One of the tabs has a record list.
That record list is also configured to open a modal when clicked.

When trying to open the second modal, I get the following error, and the modal closes (it doesn't even open the second modal)

```
Error
    at new t (vue-router.esm.js:2008:14)
    at Ue.confirmTransition (vue-router.esm.js:2124:18)
    at Ue.transitionTo (vue-router.esm.js:2068:8)
    at t.push (vue-router.esm.js:2399:10)
    at vue-router.esm.js:2811:22
    at new Promise (<anonymous>)
    at ht.push (vue-router.esm.js:2810:12)
    at Modal.vue:107:28
```

Error object details:

```json
{
    "_name": "NavigationDuplicated",
    "name": "NavigationDuplicated",
    "message": "Navigating to current location (\"/ns/crm/pages/312917135724642308?recordID=343516337831702534&recordPageID=312917085392994308\") is not allowed"
}
```

### Expected Behavior

The modal opens the second record clicked

**Comment by skamensky** [comment]
This seems to be related:

https://stackoverflow.com/a/58747480/4188138

**Comment by Fajfa** [comment]
Hmm, can't seem to replicate.
I tried on 2023.3.3
I have the same scenario as you described, will try some more different scenarios to see if i can replicate.

If you wish you can create an account on [latest](https://latest.cortezaproject.org/), and reproduce it there.
You can send me a DM/email when you do so i can give you the permission to create/build namespaces

**Comment by skamensky** [comment]
I think I know what the difficulty you're having in reproducing it. The bug only comes up when there are many related records.

In this video you can see when I have one related record, the bug does not occur. When I have 71 related records, the bug is sucessfully reproduced. Let me know if this doesn't work for you, and I'll email you for permissions on https://latest.cortezaproject.org/


https://github.com/cortezaproject/corteza/assets/19151369/00f7d272-6aa2-456c-919f-8d436c290684

**Comment by skamensky** [comment]
@Fajfa ,

Let me know what you want me to do...

**Comment by Fajfa** [comment]
@skamensky
One thing you can confirm is if this happened on 2023.3.2 aswell or not.
Otherwise  i'll be investigating further during this week.
Hopefully it can be fixed by next release

**Comment by skamensky** [comment]
@Fajfa ,

I just tested on `2023.3.2` as well. Confirmed that the issue is present there as well.

**Comment by skamensky** [comment]
@Fajfa ,

Did you still want me to reproduce this in local corteza? It should be pretty simple to reproduce with many records.

**Comment by Fajfa** [comment]
@skamensky Not for now, we found some issues regarding modals that could fix this.

**Comment by Fajfa** [comment]
Fixed with this: https://github.com/cortezaproject/corteza/pull/1333
Will be included in 2023.3.4 released this week


Another note for this issue, using a tabs block with record lists in the tabs loads all of them on the initial load
That means if you can afford to have the record list as filters instead of different tabs, it will greatly improve performance
